Week-5/Assignment5.1.js

```diff
@@ -0,0 +1,44 @@
+//async and await function
+function doTask1(value) {
+    return new Promise((resolve, reject) => {
+      if (value === "Task1") {
+        resolve(value);
+      } else {
+        reject(value);
+      }
+    });
+  }
+  
+  function doTask2(response) {
+    return new Promise((resolve, reject) => {
+      if (response === "Task1") {
+        resolve("Task2");
+      } else {
+        reject("Task2");
+      }
+    });
+  }
+  
+//generator function
+  function* doTask3() {
```

Comment:
create a more functional generator function, which should contain at least 3 yield and one return.

Reply:
Adde more functions to the generator function